utest/libdoc/test_libdoc.py

```diff
@@ -221,6 +229,18 @@ def test_TypesViaKeywordDeco(self):
         def test_DynamicLibrary_json(self):
             run_libdoc_and_validate_json('DynamicLibrary.json')
 
+        def test_DataTypesLibrary_json(self):
+            run_libdoc_and_validate_json('DataTypesLibrary.json')
+
+        def test_DataTypesLibrary_xml(self):
+            run_libdoc_and_validate_json('DataTypesLibrary.xml')
+
+        def test_DataTypesLibrary_py(self):
+            if PY_VERSION[0] == 3 and PY_VERSION[1] >= 7:
+                run_libdoc_and_validate_json('DataTypesLibrary.py')
```

Comment:
I'd prefer creating the test only when it's applicable and comparing version as a tuple:

```python
if PY_VERSION > (3, 7):
    def test_DataTypesLibrary_py(self):
        ...
```